src/tests/v1/test_production_locations.py

```
@@ -793,3 +722,18 @@ def test_claim_date_gte_greater_than_claim_date_lt(self):
             error['detail'],
             "The 'claimed_at_gt' must be less than or equal to 'claimed_at_lt'."
         )
+
+    def test_production_locations_country(self):
+
+        response = requests.get(
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)